data/json/npcs/godco/NPC_Helena_Misinter.json

```diff
@@ -0,0 +1,280 @@
+        ],
+        "topic": "TALK_DONE"
+      }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Helena_machineFood",
+    "dynamic_line": "A generous soul, are we?  That's not what I was expecting, but I'll see to it.",
+    "responses": [
+      {
+        "text": "Thank you.",
+        "effect": [
+          { "u_lose_var": "godco_corrie_machine", "type": "mission", "context": "completed" },
+          { "u_add_var": "godco_machine_reward", "type": "mission", "context": "reward", "value": "ration" },
+          { "u_add_var": "unclaimed_machine_reward", "type": "mission", "context": "reward", "value": "yes" }
+        ],
+        "topic": "TALK_DONE"
+      }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Helena_Contact_FreeMerch",
+    "dynamic_line": "Yes, I was just informed of that.  I also heard that she found a large group of people living out in an old evacuation center, who consider themselves merchants.  That would be a good place to resume our spread of the Truth, perhaps we could find others who were faithful before <the_cataclysm>.",
+    "responses": [
+      {
+        "text": "I presume you want me to pay a visit?",
+        "condition": {
+          "and": [
+            { "not": { "u_has_var": "u_met_Gavin", "type": "general", "context": "meeting", "value": "yes" } },
+            { "not": { "u_has_var": "found_refugee_center", "value": "yes" } }
+          ]
+        },
+        "topic": "TALK_GODCO_Helena_Accept_Contact_FreeMerch"
+      },
+      {
+        "text": "Oh, I've met them before!",
+        "condition": {
+          "or": [
+            { "u_has_var": "u_met_Gavin", "type": "general", "context": "meeting", "value": "yes" },
+            { "u_has_var": "found_refugee_center", "value": "yes" }
+          ]
+        },
+        "topic": "TALK_GODCO_Helena_Accept_Contact_FreeMerch"
+      },
+      { "text": "Let's change the subject.", "topic": "TALK_GODCO_Helena_2" },
+      { "text": "Good luck with that.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Helena_Accept_Contact_FreeMerch",
+    "dynamic_line": "Yes, you seem like a good fit for the job.  I'll pay you a small sum for getting some information about them.",
+    "responses": [
+      {
+        "text": "Alright, I'll go and see what's up.",
+        "effect": { "add_mission": "MISSION_GODCO_HELENA_VISIT_MERCHANTS" },
+        "topic": "TALK_DONE"
+      },
+      { "text": "Let's change the subject.", "topic": "TALK_GODCO_Helena_2" },
+      { "text": "Good luck with that.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Helena_Tell_FreeMerch",
+    "dynamic_line": "Really?  I'd pay a small sum for information about them.",
+    "responses": [
+      {
+        "text": "Well, they're…",
+        "effect": [ { "u_spawn_item": "icon", "count": 5 }, { "u_assign_activity": "ACT_EXPLAINING", "duration": "30 minutes" } ],
```

Comment:
Can't you use `ACT_SOCIALIZE` or `ACT_WAIT_NPC` here? A new act needs C++ bits too.

Reply:
I've changed it.